sass/themes/cr/2017/components/navigation/_expanded-footer.scss

```diff
@@ -0,0 +1,292 @@
+// Expanded Footer
+//
+// Expanded footer (aka TURBOFOOTER™) for the CR.com site in 2018
+// 
+// Markup: expanded-footer.twig
+//
+// Style guide: 185.5
+
+footer.footer--expanded {
+  
+  background: #f2f2f2;
+  padding: 45px 18px 40px;
+
+  span.lg-breakpoint {
+    display: none;
+    visibility: hidden;
+  }
+
+  @include breakpoint($screen-lg) {
+    padding: 65px 18px 60px;
+
+    span.lg-breakpoint {
+      display: block;
+    }
+  }
+
+  // Social Links
+  .footer__social-links {
+    margin: 0 0 40px;
+    ul li {
+      margin: 5px 8px 0;
+      a {
+        width: 52px;
+        height: 52px;
+        background: none;
+      }
+    }
+
+    @include breakpoint($screen-md) { 
+      ul li {
+        margin: 5px 10px 0;
+        a {
+          width: 75px;
+          height: 75px;
+        }
+      }
+    }
+
+    @include breakpoint($screen-lg) {
+      margin: 0 0 75px;
+    } 
+  }
+
+  .footer__copyright {
+    margin-top: 10px;
+
+    @include breakpoint($screen-md) {
+      margin-top: 15px;
+    }
+
+    @include breakpoint($screen-lg) {
+      margin-top: 10px;
+    }
+    
+    p {
+      color: $colour-black;
+      font-size: 18px;
+      line-height: 29px;
+    }
+  } 
+
+  .footer__branding {
+    margin-top: 35px;
+
+    a {
+      width: 85px;
+      height: 85px;
+    }
+    
+    @include breakpoint($screen-md) {
+      margin-top: 35px;
+    }
+    
+    @include breakpoint($screen-lg) {
+      margin-top: 60px;
+    }
+  }
+
+  .region.region-footer.cr-footer {
+    @include container();
+  }
+
+  ul.menu.menu--level-0:not(.menu--level-1) {
+    width: 100%;
+    display: -webkit-flex;
```

Comment:
I thought the 'autoprefixer' sorts this out for us so we don't define all browsers alternatives

Reply:
I'll see what I can remove; I was having iOS issues and I think it might've been overlooking a few of these rules, so I ended up just wanging the lot in there

Reply:
It's working now I've fixed the Postcss job, but it's including the webkit-box rule that what was screwing up iOS8 for me. 

As such, I'll just leave this as manual adding the vendor prefixes that actually play nicely together :)